src/sentry/scripts/tsdb/cmsketch.lua

```diff
@@ -201,7 +201,7 @@ function Sketch:coordinates(value)
 end
 
 function Sketch:exists()
-    return redis.call('EXISTS', self.estimates, self.index) > 0
+    return redis.call('EXISTS', self.index)
```

Comment:
I forgot that the index will always exist if there are contents, so this is a little simpler now.